src/app/pages/LandingPage.svelte

```diff
@@ -1,56 +1,189 @@
 <script lang="ts">
   import Upload from "../components/Upload.svelte";
-  import BCP47Tag from "../components//BCP47Tag.svelte";
+  import BCP47Tag from "../components/BCP47Tag.svelte";
 </script>
 
+<style>
+  :global(body) {
+    display: flex;
+
+    min-height: 100vh;
+    margin: 0;
+    padding: 0;
+
+    flex-flow: column nowrap;
+  }
+
+  :global(:root) {
+    --body-max-width: 960px;
+
+    background-color: var(--white);
+
+    scroll-behavior: smooth;
+  }
+
+  main {
+    flex: 1;
+    --quick-start-max-width: calc(var(--body-max-width) / 2);
+
+    margin-bottom: 2rem;
+  }
+
+  .masthead {
+    --wave-height: 322px;
+    --masthead-content-height: 20rem;
+    --masthead-background-color: #09f;
+
+    width: 100vw;
+    padding-bottom: var(--wave-height);
+
```

Comment:
Some more padding on the top would provide a little needed whitespace.


Current:
<img width="909" alt="image" src="https://user-images.githubusercontent.com/38014509/95919532-fdc25d00-0d6a-11eb-9581-e985d9f5a581.png">

Figma:
<img width="872" alt="image" src="https://user-images.githubusercontent.com/38014509/95919543-02871100-0d6b-11eb-9a66-5b7065c26d45.png">